runtime/testnet/src/lib.rs

```diff
@@ -984,7 +988,7 @@ impl_runtime_apis! {
 
 		fn dispatch_benchmark(
 			config: frame_benchmarking::BenchmarkConfig
-		) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
+		) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, alloc::string::String> {
```

Comment:
Missing `use frame_system_benchmarking::extensions::Pallet as SystemExtensionsBench;` in the benchmarks trait implementation. Closest I could place the comment.

Reply:
resolved in https://github.com/r0gue-io/pop-node/pull/447